solidity/dashboard/src/rewards-allocation/rewards.json

```diff
@@ -23251,5 +23251,2692 @@
         ]
       }
     }
+  },
+  "0x4121a37846603bdaede65f1cde2f7cb06df005cf4c5353310c03158ce5a05e59": {
```

Comment:
Looks like this address is different than the one in https://github.com/keep-network/keep-ecdsa/pull/754/files#diff-b0b0b53f7e3097714e71488d63b58e9590f59924393167b404f3f69c10cbd254R24636 in `output-merkle-objects.json`